cdap-watchdog/src/main/java/co/cask/cdap/logging/write/FileMetaDataManager.java

```diff
@@ -194,6 +199,10 @@ public Integer apply(DatasetContext<Table> ctx) throws Exception {
     });
   }
 
+  private String getLogPartition(byte[] rowKey) {
+    return Bytes.toString(Arrays.copyOfRange(rowKey, ROW_KEY_PREFIX_END.length, rowKey.length));
```

Comment:
Bytes has a version with offset and length. or is there some other reason you're copying?